torch/csrc/jit/runtime/register_ops_utils.cpp

```diff
@@ -28,6 +28,23 @@ int listIndex<at::Tensor>(Stack& stack) {
   return 0;
 }
 
+template <>
```

Comment:
You don't need those specializations I think? if you see the templated version of it, it's exactly the same with what you implemented here https://github.com/pytorch/pytorch/blob/7021635d613a212e7fb892670eba3885a15f1a20/torch/csrc/jit/runtime/register_ops_utils.h#L281

Reply:
It's not actually - it will fail the compilation:

```
/usr/include/c++/8/bits/predefined_ops.h:241:17: error: no match for ‘operator==’ (operand types are ‘c10::impl::ListElementReference<std::__cxx11::basic_string<char>, __gnu_cxx::__normal_iterator<c10::IValue*, std::vector<c10::IValue> > >’ and ‘const std::__cxx11::basi
c_string<char>’)
  { return *__it == _M_value; }
```

Another way that may work is to change the default template function, change the 3rd argument to

```
[&] (const T& b) {
  return elem == b;
}
```

But I am unsure if this will create any performance issue.